### PR TITLE
add latest tag for release docker image

### PR DESCRIPTION
**Pull request by dhuangnm**
Need to tag the latest for the release docker image

### CODE REVIEW

File: .github/actions/nm-build-docker/action.yml [hunk 1]
```diff
@@ -27,6 +27,11 @@ runs:
              --build-arg build_type=${{ inputs.build_type }} \
              --build-arg build_version=${{ inputs.build_version }} \
              --target vllm-openai . || status=$?
+      if [ ${status} -eq 0 ] && [[ "${build_type}" = "release" ]]; then
+          echo "Also tag image for release build as latest"
+          docker image tag ghcr.io/neuralmagic/nm-vllm-openai:${{ inputs.docker_tag }} ghcr.io/neuralmagic/nm-vllm-openai:latest || ((status+=$?))
```
Comment (dbarbuzzi) [inline .github/actions/nm-build-docker/action.yml:32]: It seems like the resulting command would be expanded as below; is that correct? It seems off...

```shell
docker image tag ghcr.io/neuralmagic/nm-vllm-openai:ghcr.io/neuralmagic/nm-vllm-openai:latest ghcr.io/neuralmagic/nm-vllm-openai:latest
```

File: .github/actions/nm-build-docker/action.yml [hunk 2]
```diff
@@ -27,6 +27,11 @@ runs:
              --build-arg build_type=${{ inputs.build_type }} \
              --build-arg build_version=${{ inputs.build_version }} \
              --target vllm-openai . || status=$?
+      if [ ${status} -eq 0 ] && [[ "${build_type}" = "release" ]]; then
```
Comment (andy-neuma) [inline .github/actions/nm-build-docker/action.yml:30]: nit: we've been using all caps elsewhere, e.g NIGHTLY or RELEASE or REMOTE.

**Review by andy-neuma — APPROVED** [review]
thanks

File: .github/actions/nm-build-docker/action.yml (hunk 2 above)
Comment (dhuangnm) [reply .github/actions/nm-build-docker/action.yml]: nice catch, updated

**Pull request merged**